locale/zh/messages.json

```diff
@@ -279,7 +279,7 @@
   "Disk": "磁盘",
   "Disk (MiB)": "磁盘 (MiB)",
   "Disk Allocation": "磁盘分配",
-  "Disk Space (MiB)": "磁盘空间 (MiB)",
+  "Disk Space (Mib)": "",
```

Comment:
Not really sure the right way to handle this, but this feels like a weird change to lose the translation for?

Reply:
Not sure why this was changed.